doc/testing.html

```diff
@@ -363,12 +363,12 @@ <h2 id="test-suite-control">Test suite control</h2>
 concurrency level to 1 and the timeout factor to 8. This is equivalent
 to setting <code>JTREG_JOBS=1 JTREG_TIMEOUT_FACTOR=8</code>, but using
 the keyword format means that the <code>JTREG</code> variable is parsed
-and verified for correctness, so <code>JTREG="TMIEOUT_FACTOR=8"</code>
-would give an error, while <code>JTREG_TMIEOUT_FACTOR=8</code> would
+and verified for correctness, so <code>JTREG="TIMEOUT_FACTOR=8"</code>
```

Comment:
this was a deliberate typo

Reply:
Indeed, these two were intentional to demonstrate the error handling of the JTREG make variable.

Reply:
Well that's embarrassing.. Thank you for the catch!

Fixed (reverted).